themes/default/content/blog/kubecon-na-2021-roundup/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Kubecon NA 2021 Roundup"
+
+date: 2021-11-08T22:50:38Z
+
+draft: false
+
+meta_desc: Kat Cosgrove's recording roundup of the trends, updates, and gems from Kubecon NA 2021!
+
+meta_image: meta.jpg
+
+authors:
+    - kat-cosgrove
+
+tags:
+    - kubecon
+    - kubernetes
+    - conferences
+---
+
+Kubecon North America is over, but the recordings are now online! Every talk you wanted to see and couldn’t is available on YouTube, so here’s some highlights -- cloud native trends, updates from projects and SIGs, and a few of my favorite talks!
+
+<!--more-->
+
+Kubecon North America is over, but the recordings are now online! Every talk you wanted to see and couldn’t is available on YouTube, so here’s some highlights from me -- cloud native trends, updates from projects and SIGs, and a few of my favorite talks!
+
+#### SBOM is Coming: Why You Should Care and How You Can Help - Frederick Kautz & Allan Friedman
+
+{{< youtube "4wUejdZ6KHM?rel=0" >}}
+
+A major trend throughout the conference was supply chain security and SBOMs. There were quite a few talks on these subjects, but Frederick Kautz and Allan Friedman give a great overview of exactly what an SBOM is, and why it’s important.
+
+#### Notary: State of the Container Supply Chain - Justin Cormack, Docker & Steve Lasker, Microsoft
+
+{{< youtube "hpcQJ21mn_g?rel=0" >}}
+
+On supply chain security, Steve Lasker presents updates to Notary v2. It’s changed quite a bit since v1 and there are more changes on the horizon. With the rising concerns surrounding supply chain security, I expect Notary and tools like it to be increasingly important. No worries if you aren’t already familiar with Notary -- Steve explains it well.
```

Comment:
```suggestion
On supply chain security, Steve Lasker presents updates to Notary v2. It’s changed quite a bit since v1 and there are more changes on the horizon. With the rising concerns surrounding supply chain security, I expect Notary and tools like it to be increasingly important. No worries if you aren’t already familiar with Notary&mdash;Steve explains it well.
```